modules/farhan/models/index.js

```diff
@@ -6,11 +6,46 @@ class FarhanModel{
     this.dbService = new DBService()
   }
 
-  async index(){
-    const query = `SELECT * FROM ${this.table} WHERE is_deleted=0`;
+  async index(offset = 0, limit = 10, minAge, maxAge, search, sort_by = 'id', order = 'DESC'){
+    // const query = `SELECT * FROM ${this.table} WHERE is_deleted=0`;
+    let query = `SELECT * FROM ${this.table} WHERE is_deleted=0`;
+
+    if(minAge){
+      query += ` AND age >= ${minAge}`
+    }
+
+    if(maxAge){
+      query += ` AND age <= ${maxAge}`
+    }
+
+    if(search){
+      query += ` AND name like '%${search}%'`
```

Comment:
`like` is part of query. Use an uppercase word for this. Change `like` to `LIKE`